freqtrade/optimize/optimize_reports.py

```diff
@@ -58,21 +66,46 @@ def generate_text_table(data: Dict[str, Dict], stake_currency: str, max_open_tra
                     floatfmt=floatfmt, tablefmt="pipe")  # type: ignore
 
 
-def generate_text_table_sell_reason(data: Dict[str, Dict], results: DataFrame) -> str:
+def generate_text_table_sell_reason(
+    data: Dict[str, Dict], stake_currency: str, max_open_trades: int, results: DataFrame
+) -> str:
     """
     Generate small table outlining Backtest results
     :param data: Dict of <pair: dataframe> containing data that was used during backtesting.
     :param results: Dataframe containing the backtest results
     :return: pretty printed table with tabulate as string
     """
     tabular_data = []
-    headers = ['Sell Reason', 'Count', 'Profit', 'Loss', 'Profit %']
+    headers = [
+        "Sell Reason",
+        "Sell Count",
+        "Wins",
+        "Losses",
+        "Avg Profit %",
+        "Cum Profit %",
+        f"Tot Profit {stake_currency}",
+        "Tot Profit %",
+    ]
     for reason, count in results['sell_reason'].value_counts().iteritems():
         result = results.loc[results['sell_reason'] == reason]
         profit = len(result[result['profit_abs'] >= 0])
         loss = len(result[result['profit_abs'] < 0])
         profit_mean = round(result['profit_percent'].mean() * 100.0, 2)
-        tabular_data.append([reason.value, count, profit, loss, profit_mean])
+        profit_sum = round(result["profit_percent"].sum() * 100.0, 2)
+        profit_tot = result["profit_abs"].sum()
+        profit_percent_tot = round(result["profit_percent"].sum() * 100.0 / max_open_trades, 2)
```

Comment:
I made a very small change to this section to align quotes to be identical with the lines prior to this.

While i don't really care which quotes we use - i think within "the same area" we should try to be consistent when doing the same thing.

Since it's VERY nitpicky, i made the change myself - hope you don't mind.

Reply:
Thanks for making the change, I'll keep it in mind to stick with single quotes in the future.